README.md

```diff
@@ -1,5 +1,5 @@
-Takeaway Challenge
-==================
+# Takeaway Challenge
```

Comment:
Good, clear README.
Could include more information about the user stories fulfilled, your approach and a diagram of your application design.